doc/source/tutorials/basic_usage.rst

```diff
@@ -79,11 +80,81 @@ we must we the full length string again:
 
 ``martinize2 -f protein.pdb -o topol.top -x cg_protein.pdb -ss HHHHHCCCCC``
 
+Other basic features of martinize2
+==================================
 
+Side chain fixing
+-----------------
 
+One important addition for the generation of correct Martini protein topologies is side chain fixing.
+Side chain fixing is essential for ensuring correct sampling of side chain dynamics, and involves adding
+additional bonded terms into the structure of the protein, relating to the angles and dihedrals around
+side chain and backbone atoms. For further information on the background and motivation for these terms,
+please read the paper by `Herzog et. al <https://pubs.acs.org/doi/full/10.1021/acs.jctc.6b00122>`_.
 
+Building on the command we wrote above, side chain fixing is added into martini topologies through the
+`-scfix` flag:
 
+``martinize2 -f protein.pdb -o topol.top -x cg_protein.pdb -ff martini3001 -dssp -scfix``
 
+**PLEASE NOTE**: Side chain fixes are *essential* for martini3 proteins. Currently martinize2
+(version 0.11.0) *does not* add side chain fixing parameters automatically, and so `-scfix`
+must explictly be given as an argument. This will be fixed in future version of martinize2.
 
+Secondary and tertiary structure considerations
+-----------------------------------------------
+
+The examples given on this page show how to generate basic coarse grained topologies for the
+martini force field using martinize2. Martinize2 has many further features to
+transform your simulation from a physically naive coarse grained model to one that really
+reproduces underlying atomistic behaviour. One of the most important considerations in this
+is how to treat secondary structure in the absence of hydrogen bonding. Two such methods
+are described in both the
+`Martini protein tutorial <https://cgmartini.nl/docs/tutorials/Martini3/ProteinsI/>`_, which
+should be an essential route in to conducting simulations with the martini force field.
+
+We cover the documentation of these features in greater detail in the pages about
+`Elastic Networks <elastic_networks.rst>`_ and `Gō models <go_models.rst>`_.
+
+Cysteine bridges
+----------------
+
+If your protein contains cysteine bridges, martinize2 can attempt to identify linked residues
+and add correct Martini parameters between them using the `-cys` flag. When bridged residue pairs
+are identified, a constraint of length 0.24 nm will be added between the side chains of the two
+residues.
+
+The `-cys` flag can read one of two types of argments. `-cys auto` will look for pairs of residues
+within a short cutoff:
+
+``martinize2 -f protein.pdb -o topol.top -x cg_protein.pdb -ff martini3001 -dssp -scfix -cys auto``
```

Comment:
This is the default, right? It currently reads as if cysteine bridges are totally ignored if this flag is not given.

Reply:
I'm pretty sure they are though!

Reply:
Don't think so! IIRC the edges are made based on VDW radii, the cutoff value will be somewhere in make_bonds; and there *should* be a link for the interactions.

Reply:
https://github.com/marrink-lab/vermouth-martinize/blob/master/vermouth/data/force_fields/martini3001/aminoacids.ff#L1271 for the link
https://github.com/marrink-lab/vermouth-martinize/blob/master/vermouth/processors/make_bonds.py#L52 -> 0.36 +- 0.036 nm.

https://github.com/marrink-lab/vermouth-martinize/blob/master/bin/martinize2#L952 -> If the argument is 'auto' nothing special is done, 'none' removes the edges, a float adds edges.

ALSO, WHY ON EARTH IS THE DEFAULT NONE? That makes 0 sense whatsoever....
Another thing to fix in #621 :dizzy_face:

Reply:
yeah this is a pain. Basically afaict from my experience if you give `-cys auto` then it makes the correct bridges when you have them, but if you don't give them then it doesn't make them (and this is from an atomistic simulation...). I think like you say this is corrected when the default gets changed to "auto"?!